netalertx.json

```diff
@@ -0,0 +1,46 @@
+{
+        "NetAlertX": {
+                "containers": {
+                        "netalertx": {
+                                "image": "jokobsk/netalertx",
+                                "opts": [
+                                        [
+                                                "--net",
+                                                "host"
+                                        ]
+                                ],
+                                "launch_order": 1,
+                                "ports": {
+                                    "20211": {
+                                        "description": "Rockon UI Link Port. Recommended: 20211. This port has to be the same as the subsequent Web-port later in the configuration, so the admin pages can be accessed from within the Rockon page.",
+                                        "host_default": 20211,
+                                        "label": "Rockon_Web-Port",
+                                        "ui": true
+                                    }
+                                },
+                                "volumes": {
+                                        "/app/config": {
+                                                "description": "Choose a share for NetAlertX configuration. E.g.: create a Share called netalertx-config for this purpose alone.",
+                                                "label": "NetAlertX config"
+                                        },
+                                        "/app/db": {
+                                                "description": "Choose a share for NetAlertX database. E.g.: create a Share called netalertx-db for this purpose alone.",
+                                                "label": "NetAlertX db"
+                                        }
+                                },
+                                "environment": {
+                                        "PORT": {
+                                                "description": "Enter the WebUI Web-port number. Recommended: 20211. This port has to be the same as the Rockon UI Link port, configured earlier. This ensures that the admin pages for NetAlertX are accessible via Web Browser",
+                                                "label": "Web-Port",
+                                                "index": 1
+                                        }
+                                }
+                        }
+                },
+                "description": "NetAlertX is an intrusion detection system (IDS).<p>Easily visualize all your networks in one place, enhance security with real-time alerts, and enjoy seamless integration with your smart home—no matter what hardware you use.</p><p>Based on the official docker image: <a href='https://hub.docker.com/r/jokobsk/netalertx' target='_blank'>https://hub.docker.com/r/jokobsk/netalertx</a>, available for amd64, arm64, armv7 and armv6 architecture.</p>",
```

Comment:
```suggestion
                "description": "NetAlertX is an intrusion detection system (IDS).<p>Easily visualize all your networks in one place, enhance security with real-time alerts, and enjoy seamless integration with your smart home — no matter what hardware you use.</p><p>Based on the official docker image: <a href='https://hub.docker.com/r/jokobsk/netalertx' target='_blank'>https://hub.docker.com/r/jokobsk/netalertx</a>, available for amd64 and arm64 architecture.</p>",
```
Just to be consistent with Rockstor's focus on supporting those two architectures.